static/js/chart/draw_choropleth.ts

```diff
@@ -525,6 +533,7 @@ const onMapClick = (
 
 function mouseOutAction(domContainerId: string, index: number): void {
   const container = d3.select(domContainerId);
+  container.classed(HOVER_HIGHLIGHTED_CLASS_NAME, false);
```

Comment:
duplicate of line 539

Reply:
on purpose - setting the class to the container as well so other regions are made more transparent